### PR TITLE
WS-88: Cohort filtration in analytics

### PR DESCRIPTION
**Pull request by realARCHaim**
https://youtrack.raccoongang.com/issue/WS-88

### CODE REVIEW

File: rg_instructor_analytics/static/rg_instructor_analytics/js/tab-holder.js [hunk 1]
```diff
@@ -46,10 +47,13 @@ function TabHolder(tabs, course) {
...
 
     this.selectCourse = function(course) {
-        holder.course = course;
+        holder.course = course.replace(/###.*/, '');
+        holder.cohort = course.replace(/.*###/, '');
+
         for (var tabName in tabs) {
             if (tabs.hasOwnProperty(tabName)) {
                 if (tabs[tabName].isActive) {
+                    console.log(tabName)
```
Comment (cmltaWt0) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/tab-holder.js]: Remove this, please.

**Review by cmltaWt0 — APPROVED** [review]
Remove console.log and LGMT

File: rg_instructor_analytics/static/rg_instructor_analytics/js/tab-holder.js (hunk 1 above)
Comment (realARCHaim) [reply rg_instructor_analytics/static/rg_instructor_analytics/js/tab-holder.js]: Done. Thanks

**Comment by realARCHaim** [comment]
@cmltaWt0 I've done some updates to the code after the PR is created + fixed the place, suggested by you. Could you please review it again?

**Pull request closed without merging**